### ISSUE TITLE
Ender dragon and purple ray things around it go invisible on dragon death

### ISSUE THREAD

**Issue by Isuewo** [IrisShaders/Iris#706]
### What happened?

How to reproduce:
- Kill the ender dragon with shaders enabled (I tested Complementary & BSL)

Expected vanilla (or with shaders disabled) behavior 

### Screenshots

![2021-08-04_17 25 04](https://user-images.githubusercontent.com/53264315/128209469-4365c38a-24a9-4fca-8bfd-c28012aa97b1.png)

### Relevant log output

```shell
[17:25:01] [Render thread/WARN]: Iris has detected a non-wrapped render layer, it will not be rendered with the correct shader program: entity_alpha
```

### Iris and Minecraft Version

iris-mc1.17-1.1.2-pre+rev.d30e106.jar for Minecraft 1.17 & sodium-fabric-mc1.17.1-0.3.0+IRIS2-rev.485f88c.jar for Minecraft 1.17

### Are you running Sodium along with Iris?

Yes

### Operating System

Linux

### What is your GPU?

2070

### Java Version

17

**Comment by Levelowel** [comment]
Disabling the shader makes it visible:
![2021-08-04_20 23 51](https://user-images.githubusercontent.com/53264315/128234399-81e0d458-c779-4310-9169-ed8c2d5d4a2a.png)

**Comment by Levelowel** [comment]
This is Sildurs Vibrant:
![2021-08-04_20 27 28](https://user-images.githubusercontent.com/53264315/128234873-0c3aed8c-d06e-4741-99c6-6b93df136345.png)

**Comment by coderbot16** [comment]
The death beams are related to https://github.com/IrisShaders/Iris/issues/773.

The dragon itself is caused by the `entity_alpha` shader not being properly supported on 1.17, because the effect works by changing the threshold of the alpha test, but Iris puts in a hardcoded alpha test.

**Comment by Levelowel** [comment]
https://discord.com/channels/237199950235041794/823191668064911410/893675727638126603
![Screenshot_20211005_232531](https://user-images.githubusercontent.com/53264315/136104954-ab88b6a1-1967-4af1-bd34-62bc0eba331d.png)
I'm just gonna put this here for anyone else

**Comment by coderbot16** [comment]
The purple beams part has been addressed on 1.16.x in the "bulletproof state tracking" branch. The dragon going invisible part is a separate issue to 1.17.x.
